RTCPeerConnection.js

```diff
@@ -110,14 +112,38 @@ export default class RTCPeerConnection extends EventTarget(PEER_CONNECTION_EVENT
       WebRTCModule.peerConnectionRemoveStream(stream._reactTag, this._peerConnectionId);
   }
 
+  addTransceiver(source: 'audio' |'video' | MediaStreamTrack, init) {
+    return new Promise((resolve, reject) => {
+
+      let src;
+      if (source === 'audio') {
+        src = { type: 'audio' };
+      } else if (source === 'video') {
+        src = { type: 'video' };
+      } else {
+        src = { trackId: track.id };
```

Comment:
```suggestion
        src = { trackId: source.id };
```

Reply:
I changed it already. Thanks for your reply!